lib/doWhilst.js

```diff
@@ -19,7 +20,7 @@ import onlyOnce from './internal/onlyOnce';
  * passes. The function is passed a `callback(err)`, which must be called once
  * it has completed with an optional `err` argument. Invoked with (callback).
  * @param {Function} test - synchronous truth test to perform after each
```

Comment:
Yes, lots of docs to update. :sweat:   It does create a nice link to the central `AsyncFunction` docs in the rendered docs, though.